camera.js

```diff
@@ -3030,6 +3030,7 @@ app.get(['/:auth/videos/:ke','/:auth/videos/:ke/:id'], function (req,res){
         sql.query(req.count_sql,req.count_ar,function(err,count){
             r.forEach(function(v){
                 v.href='/'+req.params.auth+'/videos/'+v.ke+'/'+v.mid+'/'+s.moment(v.time)+'.'+v.ext;
+                v.subtitleHref='/'+req.params.auth+'/subtitles/'+v.ke+'/'+v.mid+'/'+s.moment(v.time)+'.'+v.ext+'.vtt';
```

Comment:
As of right now, yes, since there is no support in the database to store whether a subtitle file was created or not.  I was asking which you prefer, but the only 2 options I see are updating database schema and adding a flag if the vtt file was created or not, or could use an fs.existsSync to check if it's on disk.